Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/Xamarin.Forms.GoogleMaps.Android.csproj

```diff
@@ -154,16 +154,16 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="$(SolutionDir)\Xamarin.Forms.GoogleMaps\Xamarin.Forms.GoogleMaps.csproj">
-      <Project>{F1C8F5A0-A55D-4C7D-9737-472F7AC222D4}</Project>
-      <Name>Xamarin.Forms.GoogleMaps</Name>
-    </ProjectReference>
-  </ItemGroup>
   <ItemGroup>
     <Folder Include="Logics\" />
     <Folder Include="Extensions\" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
I think this file's change is no need.

Reply:
Actually it's required when you use `Xamarin.Forms.GoogleMaps` as submodule. If you look at [iOS csproj](https://github.com/amay077/Xamarin.Forms.GoogleMaps/blob/master/Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.iOS/Xamarin.Forms.GoogleMaps.iOS.csproj#L103), you can see that it's done in the same way as I proposed here.